client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -23,12 +23,18 @@ import FormLabel from 'components/forms/form-label';
 import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import CountedTextarea from 'components/forms/counted-textarea';
 import Banner from 'components/banner';
-import { getSeoTitleFormatsForSite, isJetpackSite, isRequestingSite } from 'state/sites/selectors';
+import {
+	getSeoTitleFormatsForSite,
+	isJetpackSite,
+	isRequestingSite,
+	getSiteSlug,
+} from 'state/sites/selectors';
```

Comment:
It might be a good idea to order those imports alphabetically as it makes them easier to scan.